### DISCUSSION TITLE
aplus-ui/components/ap-table-modal/

### DISCUSSION THREAD

**Discussion by giscus** [ollieLiu1988/aplus-ui-comments#4] [General]
# aplus-ui/components/ap-table-modal/

Aplus basic doc site.

https://aplus.aplnk.com/aplus-ui/components/ap-table-modal/

**Comment by MyMagicCode** [comment]
1. 解决取消弹框报错问题。
2. columns类型定义扩展，额外的value-type类型类型报错问题。

**Reply by MyMagicCode** [reply]
已完成
